spec/abstractops.html

```diff
@@ -1389,10 +1389,13 @@ <h1>
         1. If _parseResult_ is not a Parse Node, then
           1. Set _parseResult_ to ParseText(StringToCodePoints(_isoString_), _goal_).
           1. If _parseResult_ is a Parse Node, then
+            1. Let _foundCalendar_ be *undefined*.
             1. For each |Annotation| Parse Node _annotation_ contained within _parseResult_, do
               1. Let _key_ be the source text matched by the |AnnotationKey| Parse Node contained within _annotation_.
               1. Let _value_ be the source text matched by the |AnnotationValue| Parse Node contained within _annotation_.
-              1. If CodePointsToString(_key_) is *"u-ca"* and the ASCII-lowercase of CodePointsToString(_value_) is not *"iso8601"*, throw a *RangeError* exception.
+              1. If CodePointsToString(_key_) is *"u-ca"*, and _foundCalendar_ is *undefined*, then
+                1. Let _foundCalendar_ be CodePointsToString(_value_).
+            1. If the ASCII-lowercase of _foundCalendar_ is not *"iso8601"*, throw a *RangeError* exception.
```

Comment:
Is it possible for there to be no annotations at this point?  If so, then can't `_foundCalendar_` be `*undefined*`?

Reply:
Thanks for catching this.